AntWars/Helper/CostCalculator.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AntWars.Board.Ants;
+
+namespace AntWars.Helper
+{
+    class CostCalculator
+    {
+        /// <summary>
+        /// Berechnet die Kosten anhand der übergebenen Ameise.
+        /// </summary>
+        /// <param name="ant">Die Ameise</param>
+        public static int calculateCost(Ant ant)
```

Comment:
das sollte kompiler errors geben da du hier kein int zurück gibst, ruf die methode dann auch in der AIBase auf, bei dem buy ant.